packages/app-store/utils.ts

```diff
@@ -142,10 +142,19 @@ export function getAppFromLocationValue(type: string): AppMeta | undefined {
  * @param concurrentMeetings - from app metadata
  * @returns - true if app supports team install
  */
-export function doesAppSupportTeamInstall(
-  appCategories: string[],
-  concurrentMeetings: boolean | undefined = undefined
-) {
+export function doesAppSupportTeamInstall({
+  appCategories,
+  concurrentMeetings = undefined,
+  isPaid,
+}: {
+  appCategories: string[];
+  concurrentMeetings: boolean | undefined;
+  isPaid: boolean;
+}) {
+  // Paid apps can't be installed on team level - That isn't supported
+  if (isPaid) {
```

Comment:
Without this some states were getting inconsistent, e.g . Repeat installation of Cal.ai was allowed.